int32 poseID
string speech
bool hasUser